app/proguard-rules.pro

```diff
@@ -37,4 +37,6 @@
 -keep class com.google.android.gms.ads.identifier.AdvertisingIdClient$Info {
     java.lang.String getId();
     boolean isLimitAdTrackingEnabled();
-}
\ No newline at end of file
+}
+
+-keep class androidx.core.app.CoreComponentFactory { *; }
```

Comment:
Added to prevent this crash that I was seeing:

```
2019-06-03 11:34:03.715 1124-1124/? E/LoadedApk: Unable to instantiate appComponentFactory
    java.lang.ClassNotFoundException: Didn't find class "androidx.core.app.CoreComponentFactory" on path: DexPathList[[],nativeLibraryDirectories=[/data/app/mozilla.lockbox-_n1hXVvdHHIGsznHfM5mkg==/lib/arm64, /data/app/mozilla.lockbox-_n1hXVvdHHIGsznHfM5mkg==/base.apk!/lib/arm64-v8a, /system/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.app.LoadedApk.createAppFactory(LoadedApk.java:226)
        at android.app.LoadedApk.updateApplicationInfo(LoadedApk.java:338)
        at android.app.ActivityThread.handleDispatchPackageBroadcast(ActivityThread.java:5437)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1741)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at com.android.server.SystemServer.run(SystemServer.java:460)
        at com.android.server.SystemServer.main(SystemServer.java:296)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:838)
```